roles/horizon/tasks/main.yml

```diff
@@ -116,3 +116,9 @@
 
 - include: monitoring.yml tags=monitoring,common
   when: monitoring.enabled|default('True')|bool
+
+- name: add horizon customization modules
+  pip: name="git+https://github.com/davidcusatis/bluebox-horizon-customization#egg=bluebox-horizon-customization"
+       virtualenv="{{ openstack_source.virtualenv_base }}/horizon/"
+  notify:
+    - restart apache
```

Comment:
Thinking about this more. Instead of doing this as a task, we should instead list this repo as a `python_dependencies`, which will get installed when the `openstack-source` role creates the virtualenv.

See https://github.com/blueboxgroup/ursula/blob/master/roles/horizon/defaults/main.yml#L9 for other examples. This will ensure the content is there before the service is ever started.

For packages, we'll add it into the giftwrap manifest so the packages will have it already inside the virtualenv.

Please work with @craigtracey on a final repository location / name.